experimental/IntersectionTheory/src/Main.jl

```diff
@@ -187,19 +187,20 @@ end
 # AbstractVarietyMap
 #
 @doc raw"""
-    hom(X::AbstractVariety, Y::AbstractVariety, fˣ::Vector)
-    hom(X::AbstractVariety, Y::AbstractVariety, fˣ::Vector, fₓ)
+     hom(X::AbstractVariety, Y::AbstractVariety, fˣ::Vector, fₓ = nothing; inclusion::Bool = false, symbol::String = "x")
 
-Construct a abstract_variety morphism from `X` to `Y`, by specifying the pullbacks of
-the generators of the Chow ring of `Y`. The pushforward can be automatically
-computed in certain cases.
+Return an abstract variety morphism from `X` to `Y` by specifying the pullbacks of
+the generators of the Chow ring of `Y`. 
+
+!!! note
+    The corresponding pushforward can be automatically computed in certain cases.
 
 In case of an inclusion $i:X\hookrightarrow Y$ where the class of `X` is not
-present in the Chow ring of `Y`, use the argument `inclusion=true`.
-A copy of `Y` will be created, with extra classes added so that one can
-pushforward classes on `X`.
+present in the Chow ring of `Y`, use the argument `inclusion = true`. Then,
+a copy of `Y` will be created, with extra classes added so that one can
+pushforward all classes on `X`.
 """
-function hom(X::AbstractVariety, Y::AbstractVariety, fˣ::Vector, fₓ=nothing; inclusion::Bool=false, symbol::String="x")
+function hom(X::AbstractVariety, Y::AbstractVariety, fˣ::Vector, fₓ = nothing; inclusion::Bool = false, symbol::String = "x")
```

Comment:
According to https://docs.oscar-system.org/dev/DeveloperDocumentation/styleguide/#Unicode, we shouldn't use unicode in identifier names like the $f^x$ here

Reply:
I suggest we ignore this for now, and adjust it once Wolfram is done with his changes.

Reply:
I agree with @thofma , there is a lot of unicode in this code which makes the code much more readable.